examples/flash_attention/example_gqa_bwd_tma_reduce_varlen.py

```
@@ -183,14 +188,14 @@ def flash_bwd_prep(
                 if by * blk + i < q_current_seqlen:
-                    Delta[q_start_idx + by * blk + i, bx] = delta[i]
+                    Delta[bz, bx, by * blk + i] = delta[i]
 
     return flash_bwd_prep
 
 
 def make_dq_layout(dQ):
-    # bshd -> bhld to use tma reduction instruction
-    return T.Layout(dQ.shape, lambda b, l, h, d: [b, h, l, d])
+    # bshd -> bhsd to use tma reduction instruction
+    return T.Layout(dQ.shape, lambda l, h, d: [h, l, d])
```

Ruff (0.14.2)

Ambiguous variable name: `l`

(E741)